### DISCUSSION TITLE
FaceSwap

### DISCUSSION THREAD

**Discussion by Crowdie0406** [MoonRide303/Fooocus-MRE#193] [Ideas]
I love this more "advanced" version, my only issue is the fact that the face swap option is not there, i've been trying to keep the same character in different settings and it's so incredibly hard to just mask a face with a model.

If there is any way you could add something like the face swap mechanic or better that would be awesome.

**Comment by TanvirHafiz** [comment]
this is probably the most important thing missing and you can just fork it off from original foocus as well. this is such an important thing to miss out!
